Source/EasyNetQ/Events/ConnectionDisconnectedEvent.cs

```diff
@@ -2,5 +2,14 @@
 {
     public class ConnectionDisconnectedEvent
     {
+        public ConnectionDisconnectedEvent(string hostname, int port, string reasonText)
+        {
+            Hostname = hostname;
+            Port = port;
+            ReasonText = reasonText;
+        }
+        public string Hostname { get; }
+        public int Port { get; }
+        public string ReasonText { get; }
```

Comment:
```suggestion
        public string Reason { get; }
```

Reply:
I've updated the commit with the suggested change